x-pack/plugins/security_solution/public/flyout/document_details/shared/hooks/use_highlighted_fields.ts

```diff
@@ -111,6 +115,13 @@ export const useHighlightedFields = ({
       return acc;
     }
 
+    if (
+      field.id === CROWDSTRIKE_AGENT_ID_FIELD &&
+      !isAlertFromCrowdstrikeEvent({ data: dataFormattedForFieldBrowser })
+    ) {
+      return acc;
+    }
+
```

Comment:
could you add a one line documentation here like it's not for the other if/else conditions above? Also I feel like adding a unit test for this would be nice